### PR TITLE
Issue #3338: [UX] Make the "This is a preview. ..." message a warning and also make it persistent

### PR DESCRIPTION
**Pull request by klonos**
https://github.com/backdrop/backdrop-issues/issues/3338

### CODE REVIEW

File: core/modules/node/node.pages.inc [hunk 1]
```diff
@@ -641,6 +640,12 @@ function node_preview_banner_form(array $form, array $form_state, Node $node, $n
   // Get the view mode to render the preview in.
   $view_mode = empty($_GET['view_mode']) ? 'full' : $_GET['view_mode'];
 
+  // We render this message here instead of in node_form_preview() in order to
+  // make it persistent across page refreshes while still in preview mode, and
+  // also while switching to different display modes.
+  backdrop_set_message(t('This is a preview, and links within the page are disabled.'), 'warning');
```
Comment (herbdool) [inline core/modules/node/node.pages.inc]: This is odd phrasing with the two independent clauses together. Better to say: "This is a preview. Links within the page are disabled."

File: core/modules/node/node.pages.inc [hunk 2]
```diff
@@ -641,6 +640,12 @@ function node_preview_banner_form(array $form, array $form_state, Node $node, $n
...
+  backdrop_set_message(t('Changes are stored temporarily. Click <em>Save</em> to make your changes permanent, or <em>Back to content editing</em> to make additional changes.'), 'warning');
```
Comment (herbdool) [inline core/modules/node/node.pages.inc]: Better to phrase it: "Changes are stored temporarily. Click <em>Save</em> to make your changes permanent, or click <em>Back to content editing</em> to make additional changes." (adding second "click").

File: core/modules/node/node.pages.inc (hunk 1 above)
Comment (klonos) [reply core/modules/node/node.pages.inc]: 👍 done!

File: core/modules/node/node.pages.inc (hunk 2 above)
Comment (klonos) [reply core/modules/node/node.pages.inc]: ...also good suggestion 👍, also done 😄

**Comment by herbdool** [comment]
The message gets duplicated when I change the display mode.

**Comment by klonos** [comment]
@herbdool I have found this to be a generic problem with messages; not a problem caused by this specific PR. As an example to reproduce, in a vanilla Backdrop installation enable maintenance mode, and then visit the views listing page. I usually get the "The site is currently in maintenance mode." 3 times 😞 ...this does not work 100% of the times (browser cache might be at play too). If so, then also try to visit page other than the views listing (like the home page for example), then clear caches, then visit the views listing page, then switch to the views settings tab. Doing that, I usually get either 2 or 3 duplicate messages, at least one time.

**Comment by klonos** [comment]
...having said that, setting the `$repeat` argument in `backdrop_set_message()` to `FALSE` seems to do the trick. If this was your only concern @herbdool, care to re-test and RTBC?

**Pull request merged**